### PR TITLE
LRQA-75074 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-75074

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

File: modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase [hunk 1]
```diff
@@ -513,7 +513,7 @@ definition {
 				locale = "nl-NL");
 		}
 
-		task ("Then both keys should be present, but only the selected language 'nl-NL' should be bold") {
+		task ("Then both keys should be present, but only the selected language 'nl-NL' should be bold and translation for language key display") {
```
Comment (Sharryxue) [inline modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase]: Update _and translation for language key display_ to  _and translation in nl-NL display_

File: modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase [hunk 2]
```diff
@@ -527,6 +527,14 @@ definition {
 				key_language = "en-US",
 				key_languageKey = "text-not-bold",
 				locator1 = "LanguageOverride#BOLD_LANGUAGE_WITH_OVERRIDE");
+
+			AssertElementPresent(
```
Comment (Sharryxue) [inline modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverride.testcase]: Use exist macro.

**Review by Sharryxue — CHANGES_REQUESTED** [review]
Hello @melodywu,
Minor change required in the test, thanks.

**Comment by melodywu** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

**Comment by Sharryxue** [comment]
No changes needed.

**Comment by Sharryxue** [comment]
Rebase and resend.
https://github.com/Sharryxue/liferay-portal/pull/454

**Pull request closed without merging**